pkg/infra/pulumi_aws/index.ts.tmpl

```diff
@@ -112,7 +112,7 @@ export = async () => {
     {{range $unit := .ExecUnits -}}
 
     {{- if eq $unit.Type "fargate"}}
-    cloudLib.createEcsService("{{$unit.Name}}",{{jsonPretty $unit.Params | indent 4}} as Partial<awsx.ecs.Container>, {{- if $unit.EnvironmentVariables}} , {{jsonPretty $unit.EnvironmentVariables | indent 4}} {{end}});
+    cloudLib.createEcsService("{{$unit.Name}}",{{jsonPretty $unit.Params | indent 4}} as Partial<awsx.ecs.Container>, {{- if $unit.EnvironmentVariables}} {{jsonPretty $unit.EnvironmentVariables | indent 4}} {{else}} [] {{end}});
```

Comment:
Seems like the library can handle the `undefined` fine:
```ts
    generateExecUnitEnvVars(
        execUnitName: string,
        env_vars?: any[]
    )
```

Reply:
I'm saying the previous version should be fine since the `, ` is also included in the original if statement.

Reply:
yep, it did! nice catch.